cms/test_utils/project/placeholderapp/views.py

```diff
@@ -24,7 +25,12 @@ def _base_detail(request, instance, template_name='detail.html',
         request.toolbar.set_object(instance)
     if template_string:
         template = Template(template_string)
-        return HttpResponse(template.render(RequestContext(request=request, dict_=context)))
+        if DJANGO_1_7:
+            return HttpResponse(template.render(RequestContext(request=request, dict_=context)))
+        else:
+            from django.template.context import make_context
+            context = make_context(context, request)
+            return HttpResponse(template.render(context))
```

Comment:
I still don't understand. I'm not getting a warning for code like this which seems similar to what's happening here:
```
from django.template.base import Template
from django.template.context import RequestContext
from django.http import HttpRequest
Template('foo').render(RequestContext(HttpRequest()))
```
In addition, it looks to me like `make_context()` just wraps creates a `RequestContext` anyway, so I don't see how that would avoid that warning.